### ISSUE TITLE
displayCursor.moveToFirst()

### ISSUE THREAD

**Issue by dschuermann** [SufficientlySecure/birthday-calendar#58]
if (displayCursor != null &amp;&amp; displayCursor.moveToFirst()) {

is moveToFirst enough?
Is it possible to find name in other contacts coming after the first?

Test this!

**Comment by call-me-matt** [comment]
Good point. Adapted the FIRSTNAME resolver according to your proposal: https://github.com/SufficientlySecure/birthday-calendar/commit/77248d2b64fe4f6953f9b881f1ea1f9a83d1ea8a
